packages/oranger/oranger.2.0.0/files/compile_ranger.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
This file is in the wrong directory. It should be in `packages/oranger/oranger.2.0.1/files`

Reply:
This file was left out of the previous commit by mistake; I just pushed it.